handler/src/main/java/io/netty/handler/ssl/OpenSslEngine.java

```diff
@@ -61,6 +64,8 @@
     private static final InternalLogger logger = InternalLoggerFactory.getInstance(OpenSslEngine.class);
 
     private static final Certificate[] EMPTY_CERTIFICATES = new Certificate[0];
+    private static final X509Certificate[] EMPTY_X509_CERTIFICATES = new X509Certificate[0];
```

Comment:
We have this already in `EmptyArrays`. How about moving `EMPTY_CERTIFICATES` there, too?